docling_core/types/doc/document.py

```diff
@@ -2319,10 +2543,10 @@ def save_as_markdown(
 
     def export_to_markdown(  # noqa: C901
         self,
-        delim: str = "\n\n",  # TODO deprecate
+        delim: str = "\n\n",
         from_element: int = 0,
         to_element: int = sys.maxsize,
-        labels: set[DocItemLabel] = DEFAULT_EXPORT_LABELS,
+        labels: set[DocItemLabel] = DOCUMENT_TOKENS_EXPORT_LABELS,
```

Comment:
Changed this, as otherwise the new logic would literally filter out any non-listed labels from showing up, i.e. we would see no captions at all in the output.